src/wazuh_db/wdb_global.c

```diff
@@ -1764,12 +1768,15 @@ int wdb_global_create_backup(wdb_t* wdb, char* output) {
         snprintf(output, OS_MAXSTR + 1, "err SQLite: %s", sqlite3_errmsg(wdb->db));
     }
 
+    sqlite3_finalize(stmt);
+
     if(OS_SUCCESS == result) {
         snprintf(path_compressed, PATH_MAX, "%s.gz", path);
         result = w_compress_gzfile(path, path_compressed);
         unlink(path);
         if(OS_SUCCESS == result) {
             wdb_global_remove_old_backups();
+            snprintf(output, OS_MAXSTR + 1, "ok %s", file_name);
```

Comment:
This a great improvement!
But following the convention of the rest of the methods in Wazuh-DB, the data retrieved after the **ok** should be in JSON format.

Reply:
Changed!